pyvista/plotting/plotting.py

```diff
@@ -2798,6 +2811,8 @@ def pick_call_back(picker, event_id):
             if callback is not None:
                 callback(self.picked_cells)
 
+            # TODO: Deactivate selection tool
```

Comment:
How?

Reply:
Here's some pieces of code from another project that uses pyvista.

Basically you just want to just want to enable the trackball style again

```python
self.parent.vtk_widget.enable_cell_picking(mesh,
					   callback=self._callback)
# start rectangular selection
self.iren.GetInteractorStyle().StartSelect()

def _callback(self):
    refine_mask = picked_cells.cell_arrays['orig_extract_id']

    # disable cell picking
    self.enable_trackball_style()
```

Reply:
Awesome, I'll try this out. Thanks!

Reply:
Forgot this was meant to be called from a PyQt widget.  If ``enable trackball style`` is called from a callback, it will crash the kernel.  Instead, have it call a signal:

Add this to the QtInteractor Class
```python
class QtInteractor():
   """blablabla"""
   self.enable_trackball_style_signal = pyqtSignal()

   def __init__(self, some_kwargs...):
      # code...
      self.enable_trackball_style_signal.connect(self.enable_trackball_style)
      # more code...

```

Then call ``self.enable_trackball_style_signal`` instead of ``self.enable_trackball_style``

Reply:
What about when using just the `Plotter` class and not the `BackgroundPlotter`?

```py
p = pv.Plotter(notebook=False)
p.add_mesh(poly)
p.enable_cell_picking()
p.show()
```

Reply:
I can't figure out how to do this. I'm settling for having the text in the corner that tells the user to press R to toggle the selection tool